docs/user-guide/how_to_use_ssh_plugin.md

```diff
@@ -0,0 +1,113 @@
+Note:
+* `DEFAULT_PRIVATE_KEY` and `DEFAULT_PUBLIC_KEY` are not fully listed for they are too long. Please refer to the examples
+behind for cases.
+* Volcano is not responsible for the validation of `ssh-key-file-path`. So please guarantee it correct by users.
+
+## Examples
+```yaml
+apiVersion: batch.volcano.sh/v1alpha1
+kind: Job
+metadata:
+  name: mpi-job
+spec:
+  minAvailable: 3
+  schedulerName: volcano
+  plugins:
+    ssh: []   ## SSH plugin register
+    svc: []
+  tasks:
+    - replicas: 1
+      name: mpimaster
+      template:
+        spec:
+          containers:
+            - command:
+                - /bin/bash
+                - -c
+                - |
+                  mkdir -p /var/run/sshd; /usr/sbin/sshd;
+                  MPI_HOST=`cat /etc/volcano/mpiworker.host | tr "\n" ","`;
+                  sleep 10;
+                  mpiexec --allow-run-as-root --host ${MPI_HOST} -np 2 --prefix /usr/local/openmpi-3.1.5 python /tmp/gpu-test.py;
+                  sleep 3600;
+              image: lyd911/mindspore-gpu-example:0.2.0
+              name: mpimaster
+              ports:
+                - containerPort: 22
+                  name: mpijob-port
+              workingDir: /home
+          restartPolicy: OnFailure
+    - replicas: 2
+      name: mpiworker
+      template:
+        spec:
+          containers:
+            - command:
+                - /bin/bash
+                - -c
+                - |
+                  mkdir -p /var/run/sshd; /usr/sbin/sshd -D; 
+              image: lyd911/mindspore-gpu-example:0.2.0
+              name: mpiworker
+              resources:
+                limits:
+                  nvidia.com/gpu: "1"
+              ports:
+                - containerPort: 22
+                  name: mpijob-port
+              workingDir: /home
+          restartPolicy: OnFailure
+```
+Note:
+* This example will create a mpi job with 1 `master` and 2 `workers`.
+* Because the `SVC` plugin is enabled, you can get all the hosts in any pod as follows.
+```
+[root@mpi-job-master-0 /]# cat /etc/volcano/mpimaster.host
+mpi-job-mpimaster-0.mpi-job
+[root@mpi-job-master-0 /]# cat /etc/volcano/mpiworker.host 
+mpi-job-mpiworker-0.mpi-job
+mpi-job-mpiworker-1.mpi-job
+```
+* You can sign in other hosts in `master` pod as follows.
+```
+[root@mpi-job-master-0 /]# ssh root@mpi-job-mpiworker-0
```

Comment:
```suggestion
[root@mpi-job-master-0 /]# ssh mpi-job-mpiworker-0
```
Because the two container users are the same, there is no need to specify

Reply:
I'm ok about that.